src/Calculator.macOS/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,36 @@
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following 
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("$projectname$")]
```

Comment:
Looks like it did not get changed, nor in the rest of the file... :) A bug for the templates.